### PR TITLE
Vic DUELS-11 Adding dependency bash script

### PR DESCRIPTION
**Pull request by vlondon2**
[DUELS-{####}](https://server-duels.atlassian.net/browse/DUELS-{####})

- [ ] Bug
- [X] Feature
- [X] Dependency
- [ ] Has Dependencies

### Previous Behavior
No script to download python3 and pip

### New Behavior
Script exists that downloads python3 and pip

### Dependencies
none

### CODE REVIEW

File: get_dependencies.sh
Comment (domhankle) [inline get_dependencies.sh:1]: I think this an accidental duplicate file.

File: scripts/get_dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (domhankle) [inline scripts/get_dependencies.sh]: This isn't a wrong use of the `shebang`, but there is a more portable one we should use to better support other developers machines:

`#!/usr/bin/env bash`

**Review by domhankle — CHANGES_REQUESTED** [review]
This looks good, just some file cleanup and one minor tweak.

File: get_dependencies.sh
Comment (vlondon2) [reply get_dependencies.sh:1]: I didn't realize I had to use git mv instead of mv lol

**Review by domhankle — APPROVED** [review]
Nice work. LGTM!

**Pull request merged**